UNRELEASED.md

```diff
@@ -2,6 +2,10 @@
 
 > Place your changes below this line.
 
+**Breaking**:
+
+- Bump react-native to `0.60.4`
```

Comment:
Is this breaking for all our packages?

Reply:
Yes because some a11y pros where removed. It will still work but the props are now ignored, so I believe we should mark is as breaking for all packages